Processor cores

Misko-2083 opened the pull request:
If there are multiple processor cores it will be displayed at the end.
Eg: Pentium 77 @55GHz x256
Assuming 256 is the number of cores :)

linuxlite commented:
Thank you @Misko-2083

sysdrum commented:
Well something just broke me.

Misko-2083 commented:
Something? What is the error message?

sysdrum commented:
Fails to loud anything in box but no traceback.
On Jun 1, 2015 4:12 PM, "Милош Павловић" <notifications@github.com> wrote:

> Something? What is the error message?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/linuxlite/litecontrolcenter/pull/2#issuecomment-107713694>
> .
>

Misko-2083 commented:
Run
python /usr/share/litecc/litecenter.py
Strange, works in LL 2.4 and Mint 17

sysdrum commented:
I have the box pops up but nothing shows up. I revert the processed changes
and everything shows back up.
On Jun 2, 2015 9:20 AM, "Милош Павловић" <notifications@github.com> wrote:

> Run
> python /usr/share/litecc/litecenter.py
> Strange, works in LL 2.4 and Mint 17
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/linuxlite/litecontrolcenter/pull/2#issuecomment-107971057>
> .
>

linuxlite commented:
We'll be running lite-controlpanel with the python3 command. 
On 3/06/2015 2:22 am, David Johns <notifications@github.com> wrote:
>
> I have the box pops up but nothing shows up. I revert the processed changes
> and everything shows back up.
> On Jun 2, 2015 9:20 AM, "Милош Павловић" <notifications@github.com> wrote:
>
> > Run
> > python /usr/share/litecc/litecenter.py
> > Strange, works in LL 2.4 and Mint 17
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <https://github.com/linuxlite/litecontrolcenter/pull/2#issuecomment-107971057>
> > .
> >
>
> —
> Reply to this email directly or view it on GitHub.

sysdrum commented:
I am running it python3 which is my default for python on the 64. But it is
also doing the same thing on the 32 bit running with the python3 command.
On Jun 2, 2015 9:33 AM, "Linux Lite" <notifications@github.com> wrote:

> We'll be running lite-controlpanel with the python3 command.
> On 3/06/2015 2:22 am, David Johns <notifications@github.com> wrote:
> >
> > I have the box pops up but nothing shows up. I revert the processed
> changes
> > and everything shows back up.
> > On Jun 2, 2015 9:20 AM, "Милош Павловић" <notifications@github.com>
> wrote:
> >
> > > Run
> > > python /usr/share/litecc/litecenter.py
> > > Strange, works in LL 2.4 and Mint 17
> > >
> > > —
> > > Reply to this email directly or view it on GitHub
> > > <
> https://github.com/linuxlite/litecontrolcenter/pull/2#issuecomment-107971057
> >
> > > .
> > >
> >
> > —
> > Reply to this email directly or view it on GitHub.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/linuxlite/litecontrolcenter/pull/2#issuecomment-107974339>
> .
>

sysdrum commented:
Can we just do a try statement with it just in case maybe there is a bug in
python that might be causing the issue.
On Jun 2, 2015 9:36 AM, "David Johns" <sysdrum@gmail.com> wrote:

> I am running it python3 which is my default for python on the 64. But it
> is also doing the same thing on the 32 bit running with the python3 command.
> On Jun 2, 2015 9:33 AM, "Linux Lite" <notifications@github.com> wrote:
>
>> We'll be running lite-controlpanel with the python3 command.
>> On 3/06/2015 2:22 am, David Johns <notifications@github.com> wrote:
>> >
>> > I have the box pops up but nothing shows up. I revert the processed
>> changes
>> > and everything shows back up.
>> > On Jun 2, 2015 9:20 AM, "Милош Павловић" <notifications@github.com>
>> wrote:
>> >
>> > > Run
>> > > python /usr/share/litecc/litecenter.py
>> > > Strange, works in LL 2.4 and Mint 17
>> > >
>> > > —
>> > > Reply to this email directly or view it on GitHub
>> > > <
>> https://github.com/linuxlite/litecontrolcenter/pull/2#issuecomment-107971057
>> >
>> > > .
>> > >
>> >
>> > —
>> > Reply to this email directly or view it on GitHub.
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/linuxlite/litecontrolcenter/pull/2#issuecomment-107974339>
>> .
>>
>

linuxlite commented:
Go for it. 
On 3/06/2015 2:39 am, David Johns <notifications@github.com> wrote:
>
> Can we just do a try statement with it just in case maybe there is a bug in
> python that might be causing the issue.
> On Jun 2, 2015 9:36 AM, "David Johns" <sysdrum@gmail.com> wrote:
>
> > I am running it python3 which is my default for python on the 64. But it
> > is also doing the same thing on the 32 bit running with the python3 command.
> > On Jun 2, 2015 9:33 AM, "Linux Lite" <notifications@github.com> wrote:
> >
> >> We'll be running lite-controlpanel with the python3 command.
> >> On 3/06/2015 2:22 am, David Johns <notifications@github.com> wrote:
> >> >
> >> > I have the box pops up but nothing shows up. I revert the processed
> >> changes
> >> > and everything shows back up.
> >> > On Jun 2, 2015 9:20 AM, "Милош Павловић" <notifications@github.com>
> >> wrote:
> >> >
> >> > > Run
> >> > > python /usr/share/litecc/litecenter.py
> >> > > Strange, works in LL 2.4 and Mint 17
> >> > >
> >> > > —
> >> > > Reply to this email directly or view it on GitHub
> >> > > <
> >> https://github.com/linuxlite/litecontrolcenter/pull/2#issuecomment-107971057
> >> >
> >> > > .
> >> > >
> >> >
> >> > —
> >> > Reply to this email directly or view it on GitHub.
> >>
> >> —
> >> Reply to this email directly or view it on GitHub
> >> <https://github.com/linuxlite/litecontrolcenter/pull/2#issuecomment-107974339>
> >> .
> >>
> >
>
> —
> Reply to this email directly or view it on GitHub.

Misko-2083 commented:
Something like this @sysdrum ?
```python
        if info == "processor":
            try:
                processor = execute("grep 'model name' /proc/cpuinfo").split(':')[1]
                cores =  execute("grep 'cpu cores' /proc/cpuinfo").split(':')[1].strip(' ')
                if int(cores) == 1:
                   return processor
                else:
                   return "{0} x{1}".format(processor, cores)
            except:
                return execute("cat /proc/cpuinfo | grep 'model name'").split(':')[1]
```
